project-base/storefront/components/Blocks/OrderSummary/OrderSummary.tsx

```diff
@@ -58,13 +67,27 @@ export const OrderSummary: FC<OrderSummaryProps> = ({ isTransportOrPaymentLoadin
                                 )}
                             </AnimatePresence>
 
-                            {promoCodes.map((promoCode) => (
-                                <PromoCode
-                                    key={promoCode.code}
-                                    discount={promoCode.discount}
-                                    promoCode={promoCode.code}
-                                />
-                            ))}
+                            {promoCodes.length > 0 && (
+                                <>
+                                    {promoCodes.map((promoCode) => (
+                                        <PromoCode key={promoCode.code} promoCode={promoCode} />
+                                    ))}
+                                    <OrderSummaryRowWrapper>
```

Comment:
Is it possible to simplify these wrappers to show just one component?

Reply:
Not sure, we are also using the wrapper in `TransportAndPayment` component, there we drop the `OrderSummaryContent` and use custom styled div.

Reply:
well, ok then :)